purchases/src/main/kotlin/com/revenuecat/purchases/utils/OfferingImagePreDownloader.kt

```diff
@@ -26,25 +26,19 @@ import com.revenuecat.purchases.paywalls.components.common.Background
 import com.revenuecat.purchases.paywalls.components.properties.ThemeImageUrls
 
 internal class OfferingImagePreDownloader(
-    /**
-     * We check for the existance of the paywalls SDK. If so, the Coil SDK should be available to
-     * pre-download the images.
-     */
```

Comment:
Should we perhaps keep this clarifying comment?

Reply:
Oh whoops true. I moved it and forgot to bring it back. Thanks!